package/libs/argp-standalone/Makefile

```diff
@@ -45,4 +46,14 @@ define Build/InstallDev
 		$(1)/usr/lib/
 endef
 
+define Host/Install
+	$(INSTALL_DIR) $(1)/include
+	$(CP)   $(HOST_BUILD_DIR)/argp.h \
+		$(1)/include/
+	$(INSTALL_DIR) $(1)/lib
+	$(CP)   $(HOST_BUILD_DIR)/libargp.a \
+		$(1)/lib/
+endef
+
 $(eval $(call BuildPackage,argp-standalone))
+$(eval $(call HostBuild))
```

Comment:
No newline?

Reply:
yes, no newline. Is it mandatory? does OpenWrt have guidelines related to this?

Reply:
It's best practice. Both vim and nano automatically add one by default.

Reply:
ok, I will this no newline issue

Reply:
@neheb 'no newline' issue is fixed